encrypted-media-respec.html

```diff
@@ -1581,23 +1581,25 @@
         The <a def-id="monitor-cdm-algorithm"></a> algorithm MUST be run in parallel to the main event loop but not in parallel to other procedures defined
         in this specification that are also defined to be run in parallel.
       </p>
-
-      <p>
-        If a <a>MediaKeySession</a> object becomes inaccessible to the page
-        and is not <a def-id="media-key-session-closed"></a>, the User Agent
-        MUST use the CDM to close the session before User Agent state
-        associated with the <a href="#key-session">session</a> is deleted.
-      </p>
       <p>
-        A MediaKeySession object SHALL NOT be destroyed and SHALL continue to receive events if it is
+        A <a>MediaKeySession</a> object SHALL NOT be destroyed and SHALL continue to receive events if it is
         not <a def-id="media-key-session-closed"></a> and the <a>MediaKeys</a> object that created it remains accessible.
-        Otherwise, a MediaKeySession object that is no longer accessible to JavaScript SHALL NOT receive further
+        Otherwise, a <a>MediaKeySession</a> object that is no longer accessible to JavaScript SHALL NOT receive further
         events and MAY be destroyed.
       </p>
       <p class="note">
         The above rule implies that the CDM instance must not be destroyed until all <a>MediaKeys</a>
         objects and all <a>MediaKeySession</a> objects associated with the CDM instance are destroyed.
       </p>
+      <p>
+        If a <a>MediaKeySession</a> object is not <a def-id="media-key-session-closed"></a> when it becomes inaccessible to the page, the CDM SHALL close the <a href="#key-session">key session</a> associated with the object.
+      </p>
+      <p class="note">Closing the key session results in the destruction of any license(s) and key(s) that have not been explicitly stored.</p>
+      <p class="note">
+        Exactly when the key session is closed is an implementation detail, and applications SHOULD NOT rely on this behavior.
```

Comment:
s/this behavior/specific timing/ perhaps ?

Previously, there was a specification requirement that this should happen (without saying when). A specification requirement is not a guarantee, but it is a requirement.